tests-upgrade/AutoRestUpgradeTest.ps1

```diff
@@ -131,9 +129,9 @@ function CompareGeneratedCode([string]$inputSourcePath,[string]$inputTargetPath,
         $ignoreResult = IsNeedIgnore -inputFileName $targetFile.FullName -ignoreArray $targetIgnoreFileList
         if(!$ignoreResult)
         {
-            $obj = "waht2" | Select-Object -Property HashCode, Status
+            $obj = "what2" | Select-Object -Property HashCode, Status
             #if the file is not filefolder
-            if($targetFile.mode -eq '-a---')
+            if($targetFile.mode.Startswith('-a'))
```

Comment:
Please use file method instead of comparing mode string directly.

Reply:
Done